src/pocketmine/network/mcpe/protocol/ProtocolInfo.php

```diff
@@ -37,7 +37,7 @@ interface ProtocolInfo{
 	 */
 
 	/** Actual Minecraft: PE protocol version */
-	public const CURRENT_PROTOCOL = 389;
+	public const CURRENT_PROTOCOL = 390;
 	/** Current Minecraft PE version reported by the server. This is usually the earliest currently supported version. */
 	public const MINECRAFT_VERSION = 'v1.14.0';
```

Comment:
You'll need to change this to 1.14.60.

Reply:
I understand that you don't have the tools to generate this like I did, but please maintain the same standards if you change it by hand.